### ISSUE TITLE
Merging multiple single-page asciidoc files into single work?

### ISSUE THREAD

**Issue by samwilson** [gitenberg-dev/documentation#7]
Is anyone working on some sort of system for being able to track the relationship between pages in a scanned work and the pages' location in the final asciidoc file? Maybe single text files for each page, which then get merged into one (as part of the build script)? I'm not sure of the best way, but have been thinking of playing with some scripts for doing this.

The hardest thing I've found with working with proofreading Gitenberg books is the annoyance of keeping track of where one is in the `.djvu` vs the `.asciidoc`.... Any suggestions?

**Comment by eshellman** [comment]
Is this information preserved in any gutenberg txt or html  files?

Anchors, for example `[[source-page-13]]` should work, if you can figure out how to embed them

**Comment by samwilson** [comment]
No, I _think_ it's all thrown away. Maybe kept at PGDP though? I'm not sure.

The main issues are to do with paragraph breaks, and hyphenated words. Because one could e.g. just concatenate a bunch of asciidoc files (fragments, really) and it would work, but there would need to be conventions about which side of the pagebreak the full hyphenated word would be, and I think an extra line would have to be left at the top of pages on which a new paragraph is started.

Other than that, I think it'd work. Maybe. Hard to parse/format indiviudal pages, but that's no great problem.

**Comment by samwilson** [comment]
@eshellman I wasn't quite thinking yesterday I think! You're quite right, of course: anchors are the way to go. I had thought that the double-square-bracket syntax was only for BlockIds, but now I remember that it's fine inline too. So this would work:

```
Lorem ipsum dolor sit amet, consectetur [[page-23]] adipiscing elit. Donec a diam lectus. 
```

The only thing to decide, and perhaps it doesn't matter, is what side of cross-page items (hyphenated words etc.) to put the anchor on.

**Comment by eshellman** [comment]
Looking forward to your sensible recommendation.

**Comment by samwilson** [comment]
Hm "sensible" eh? You might be waiting a while... ;)

Closing this; I've sent a PR, where we can talk more.
